src/core/hid/input_converter.cpp

```diff
@@ -287,6 +287,20 @@ Common::Input::CameraStatus TransformToCamera(const Common::Input::CallbackStatu
     return camera;
 }
 
+Common::Input::NfcStatus TransformToNfc(const Common::Input::CallbackStatus& callback) {
+    Common::Input::NfcStatus nfc{};
+    switch (callback.type) {
+    case Common::Input::InputType::Nfc:
+        nfc = callback.nfc_status;
```

Comment:
```suggestion
        return callback.nfc_status;
```

and the default case can just be `return {};`